src/otx/algo/detection/atss.py

```diff
@@ -116,15 +116,17 @@ def _customize_outputs(
         for img_info, prediction in zip(inputs.imgs_info, predictions):
             if not isinstance(prediction, InstanceData):
                 raise TypeError(prediction)
-            scores.append(prediction.scores)  # type: ignore[attr-defined]
+
+            filtered_idx = torch.where(prediction.scores > self.hparams.get("best_confidence_threshold", 0.5))  # type: ignore[attr-defined]
```

Comment:
If there is no `best_confidence_threshold` here, what do you think about `warning` or `logging` that it is used as 0.5?

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3634/commits/5ede7974abc3b840c06054c54776424fb7c79240